File: C#/백준/Bronze/20499. Darius님 한타 안 함？/Darius님 한타 안 함？.cs
using System;

namespace q{
    class w{
        static void Main(){
            string[] aaa = Console.ReadLine().Split("/");
            int[] aa = Array.ConvertAll(aaa,int.Parse);
            if((aa[1]==0)||(aa[0]+aa[2]<aa[1])){
                Console.WriteLine("hasu");
            }
            else{
                Console.WriteLine("gosu");
            }
        }
    }
}

File: Java/백준/Bronze/20499. Darius님 한타 안 함？/Darius님 한타 안 함？.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String[] aa = in.next().split("/");
        int k = Integer.parseInt(aa[0]);
        int d = Integer.parseInt(aa[1]);
        int a = Integer.parseInt(aa[2]);
        if(d==0 || k+a<d){
            System.out.println("hasu");
        }
        else{
            System.out.println("gosu");
        }
    }
}